src/internal/table.rs

```diff
@@ -109,6 +109,12 @@ impl Table {
         let num_columns = self.columns.len();
         let num_rows =
             if row_size > 0 { (data_length / row_size) as usize } else { 0 };
+        if num_rows > 65536 {
```

Comment:
Thanks for the PR.  Could you just add a brief comment explaining this limit, with a link to the FAQ where this constant came from?

Reply:
Thank you for the review.

I've added a comment.